web/app/view/ReportController.js

```diff
@@ -525,5 +531,37 @@ Ext.define('Traccar.view.ReportController', {
         text: Strings.reportSpentFuel,
         dataIndex: 'spentFuel',
         renderer: Traccar.AttributeFormatter.getFormatter('spentFuel')
+    }],
+
+    stopsColumns: [{
+        text: Strings.reportDeviceName,
+        dataIndex: 'deviceId',
+        renderer: Traccar.AttributeFormatter.getFormatter('deviceId')
+    }, {
+        text: Strings.reportStartTime,
+        dataIndex: 'startTime',
+        xtype: 'datecolumn',
+        renderer: Traccar.AttributeFormatter.getFormatter('startTime')
+    }, {
+        text: Strings.positionAddress,
+        dataIndex: 'address',
+        renderer: Traccar.AttributeFormatter.getFormatter('address')
+    }, {
+        text: Strings.reportEndTime,
+        dataIndex: 'endTime',
+        xtype: 'datecolumn',
+        renderer: Traccar.AttributeFormatter.getFormatter('endTime')
+    }, {
+        text: Strings.reportDuration,
+        dataIndex: 'duration',
+        renderer: Traccar.AttributeFormatter.getFormatter('duration')
+    }, {
+        text: Strings.reportEngineHours,
+        dataIndex: 'engineHours',
+        renderer: Traccar.AttributeFormatter.getFormatter('duration')
```

Comment:
Used `duration` formatter here instead of `hours` because it is frequently less than a hour.